lib/order.rb

```diff
@@ -0,0 +1,52 @@
+class Order
+
+  attr_reader :summary, :menu
+
+  def initialize (menu_klass = Menu.new, order_summary_klass\
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)